float64 x
float64 y
float64 z
float64 v

int keypoint_number
string keypoint_name
